Not complete installation guide for Fedora

I had an issue when installing on Fedora:
```
psutil/_psutil_common.c:9:10: fatal error: Python.h: No such file or directory
        9 | #include <Python.h>
          |          ^~~~~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```
Fixed with `sudo dnf install python3-devel` which was not mentioned in `Readme`

--- a/README.md
+++ b/README.md
@@ -89,7 +89,7 @@ If you get an undefined symbol error on `gst_player_get_config` try removing `gi
 #### On Fedora
 
 ``` sh
-sudo dnf install python3 python3-virtualenv python3-pip
+sudo dnf install python3 python3-virtualenv python3-pip python3-devel
 python3 ./mach bootstrap
 ```
 

